### ISSUE TITLE
Polar Ignite

### ISSUE THREAD

**Issue by JeffCdev** [rsc-dev/loophole#18]
Hi Roscoe, I'm wondering if you're still doing any work on this.  I bought a polar ignite model, and unfortunately loophole doesn't seem to want to connect.  If you have any thoughts on what I might try, I'd appreciate it very, very much.

**Comment by rsc-dev** [comment]
What are the error messages? I did not test it with Polar Ignite.

**Comment by JeffCdev** [comment]
Hi rsc-dev, sorry about the delay in responding!  it is saying that no devices are detected.  I'm wondering if there is a different protocol buffer file for the ignite.

**Comment by rsc-dev** [comment]
To be honest it might be very difficult to tell why it does not work. I do not have Ignite, so I cannot debug communication.

**Comment by edneville** [comment]
Hello @rsc-dev, how did you do the debugging for the other devices?

**Comment by JeffCdev** [comment]
Hi rsc.dev, I found this comment on another polar github project (using Ruby rather than python) "Polar has changed the USB interface from HID to CDC ACM. I guess you can talk to the newer Polar watches on /dev/ttyACM0 instead (serial port for the device)" I'm wondering this is applicable to loophole,, and if so, what would be involved to make the switch.  Thanks!!!

**Comment by edneville** [comment]
Is that from [cmaion](https://github.com/cmaion/polar/issues/12#issuecomment-537390508)?

**Comment by JeffCdev** [comment]
Yep, still on the hunt for a way to get the data off my polar ignite!
